datahub-web-react/src/app/entity/user/type.ts

```diff
@@ -0,0 +1,13 @@
+import { EntityRelationshipsResult, EntityRelationship, Entity, CorpGroupProperties} from '../../../types.generated';
+
+export interface ExtendedEntityRelationshipsResult extends EntityRelationshipsResult {
+    relationships: Array<ExtendedEntityRelationship>
+}
+
+interface  ExtendedEntityRelationship extends EntityRelationship {
+    entity: ExtendedEntity
+}
+
+interface ExtendedEntity extends Entity {
```

Comment:
Why do we need this? Can we remove it?

Reply:
Yes, added few types `name` and `CorpGroupProertied` to get the `displayName`